packages/xo-server/src/xapi/mixins/networking.js

```diff
@@ -84,5 +84,9 @@ export default {
           ),
         ]),
     },
+
+    txChecksumming: {
+      set: (value, vif) => vif.update_other_config('ethtool-tx', String(value)),
```

Comment:
Isn't it better to remove it from `other_config` when value is `false`?

Reply:
For me, it's the same thing why should we handle this case? And why is better?

Reply:
So that we don't pollute the `other_config` with unnecessary entries.

Reply:
Are you sure that on removing the entry from `other_config`, the xapi will disable the functionality? Because, it's not mentioned in the documentation and i don't have a way to test it. https://xapi-project.github.io/xen-api/networking.html

Reply:
I don't know but that's what is assumed [here](https://github.com/vatesfr/xen-orchestra/pull/5182/files#diff-cbe526e73142a277b38a628ece1c6b57R654).

Reply:
It's assumed that the functionality is enabled when its value is `"true" or "on"`  as mentioned in the documentation but the action of enabling/disabling this functionality is a different think. In the documentation they say put `off` or `false` to disable it. they don't say simply remove it from the `other-config`. So, i don't know its behavior.

Reply:
But after the translation from `xapi-object-to-xo`, `txChecksumming` is a boolean. So it also assumes that when it's anything else than `'true'` or `'on'`, then it's disabled.

Reply:
There is what i understand from the documentation. The value to trigger the action of enabling the functionality is `on` the value to trigger the action of disabling the functionality is `off` if we don't trigger any action is `undefined`  which means the default VIF `txChecksuming`. I assumed that this functionality is disabled by default because  it's not the usual case for sending packets via the network.